Fix reproducibility by inherriting CFLAGS from the environment, so
that -ffile-prefix-map or -fdebug-prefix-map values are passed.

Index: mescc-tools/kaem/makefile
===================================================================
--- mescc-tools.orig/kaem/makefile
+++ mescc-tools/kaem/makefile
@@ -21,7 +21,7 @@ VPATH = bin:test:test/results
 all: kaem
 
 CC=gcc
-CFLAGS=-D_GNU_SOURCE -std=c99 -ggdb
+CFLAGS:=$(CFLAGS) -D_GNU_SOURCE -std=c99 -ggdb
 
 kaem: kaem.c
 	$(CC) $(CFLAGS) kaem.c variable.c ../functions/file_print.c ../functions/match.c ../functions/in_set.c ../functions/string.c ../functions/require.c ../functions/numerate_number.c -o ../bin/kaem
